fw/http.c

```diff
@@ -6782,6 +6782,18 @@ tfw_http_resp_process(TfwConn *conn, TfwStream *stream, struct sk_buff *skb)
 	 */
 	bad_req = hmresp->req;
 	tfw_http_popreq(hmresp, false);
+
+	/*
+	 * The special case - malformed response on Health Monitoring request.
+	 * In this case client is not involved so we just log-n-drop
+	 * this response without following processing.
+	 */
```

Comment:
Perhaps, something like this?
_Special case: malformed response to a Health Monitor request. There's no client involved, so just log-n-drop the response now without further processing._